controllers/flinkcluster/flinkcluster_util_test.go

```diff
@@ -107,20 +108,26 @@ func TestNewRevision(t *testing.T) {
 				JarFile:       &jarFile,
 				Parallelism:   &parallelism,
 				SavepointsDir: &savepointDir,
+				RestartPolicy: &restartPolicy,
 			},
 		},
 	}
+
+	var patched = flinkCluster.DeepCopy()
+	// RestartPolicy is not part of the revision.
+	patched.Spec.Job.RestartPolicy = nil
+
 	var collisionCount int32 = 0
 	var controller = true
 	var blockOwnerDeletion = true
-	var raw, _ = getPatch(&flinkCluster)
+	var raw, _ = getPatch(patched)
 	var revision, _ = newRevision(&flinkCluster, 1, &collisionCount)
 	var expectedRevision = appsv1.ControllerRevision{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "mycluster-fb7687bf5",
+			Name:      "mycluster-7bc87c954f",
```

Comment:
New revisions trigger cluster updates! so this value change should be treated carefully. In this case this is changed to previous value.